src/igvfd/tests/test_upgrade_assay_term.py

```diff
@@ -60,3 +60,9 @@ def test_assay_term_upgrade_9_10(upgrader, assay_term_v9):
     value = upgrader.upgrade('assay_term', assay_term_v9, current_version='9', target_version='10')
     assert value['schema_version'] == '10'
     assert set(value.get('preferred_assay_titles')) == {'10x multiome with scMito-seq'}
+
+
+def test_assay_term_upgrade_9_10(upgrader, assay_term_v10):
```

Comment:
The name of this test should be updated and I think the fixture "assay_term_v10" is missing from the changes (it seems to be causing the test failure)